ReactNativeClient/lib/services/InteropService.js

```diff
@@ -130,6 +139,15 @@ class InteropService {
 		return output;
 	}
 
+	newModuleFromPath_(options) {
```

Comment:
Any reason why this function was needed to load the modules?

Reply:
The existing `newModule_` fn would load by the input format. This was fine when there was a 1-1 mapping of input formats to output formats, but now that we have 2 possible outputs for an `enex` input, we need to be explicit with which importer we want to use.

It might make sense to simply move all the existing formatters to the `newModuleFromPath` approach, so that there's only one way to do this mapping. Would you like me to do that? If so, I'd prefer to do that refactoring in a separate PR since this one is already quite large.

Reply:
Btw I pushed a commit to update the existing function names to make this slightly more clear in the meantime.

Reply:
Ok that makes sense. No need to refactor for now, it can be done if it ever becomes necessary. If you could add the explanation above as a comment for now that would be great.